sortinghat/core/jobs.py

```diff
@@ -421,6 +421,73 @@ def _group_recommendations(recs):
     return job_result
 
 
+@django_rq.job
+def autogender(ctx, uuids=None, api_token=None):
```

Comment:
I never liked this name. I think we should use `genderize` (which is linked to the service we use).